vcpkg/consume/binary-caching.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Reduce package build times with binary caching
+description: Learn to set up a binary cache to reduce rebuild times.
+author: vicroms
+ms.author: viromer
+ms.date: 8/13/2023
+ms.prod: vcpkg
+---
+# Reduce package build times with binary caching
+
+Most ports in the [vcpkg public registry](<https://github.com/Microsoft/vcpkg>) are built from
+source. Building from source ensures maximum compatiblity by using the same environment, build
+tools, and configurations you use for your project in your dependencies. However, this  can also
+result in duplicated builds when multiple projects, developers, or machines require the same
+dependencies.
+
+Setting up a binary cache allows vcpkg to save the build output of a package in a shared location
+and reuse it when asked to build the same package with the same configuration, saving time on
+rebuilds. 
+
+Binary caches can be hosted in a variety of environments, the most basic form of binary caching is a
+directory containing zipped archives of a package's build output. More advanced scenarios include
+NuGet package feeds hosted in [GitHub Packages](<https://docs.github.com/en/packages>), [Azure
+DevOps Artifacts](<https://learn.microsoft.com/azure/devops/artifacts/start-using-azure-artifacts?view=azure-devops&tabs=nuget>),
+or other such services.
+
+Binary caching is especially effective in scenarios where dependencies and configuration
+don't change often, such as in CI or build servers. Binary packages are particularly susceptible to
+configuration changes (compiler version, compilation flags, installed tools, vcpkg-specific scripts,
+etc) which makes sharing them between developer machines difficult. See the [`vcpkg
+export`](../commands/export.md) for a different option to share built packages.
+
+See the [Binary Caching documentation](../users/binarycaching.md) to learn about the different [storage backends](../users/binarycaching.md#providers).
+
+## Set up a local binary cache
+
+### Default binary cache
+
+By default, vcpkg creates a local binary cache. This helps save time when installing the same
+package on the same machine.
+
+The location of the default binary cache depends on your operative system. 
+
+### [Windows](#tab/default-windows)
+
+Binary caching is enabled by default at the first valid location of:
+
+* The location specified by the `VCPKG_DEFAULT_BINARY_CACHE` environment variable
+* `%LOCALAPPDATA%\vcpkg\archives`
+* `%APPDATA%\vcpkg\archives`
+
+### [Non-Windows](#tab/default-non-windows)
```

Comment:
I think there's a way to synchronize these tabs across multiple pages and sections.

Also, should we use `Unix` instead of `Non-Windows`?